### PR TITLE
exif: add page

### PR DESCRIPTION
**Pull request by ThatWeirdAndrew**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

#4639

### CODE REVIEW

**Review by sbrl — CHANGES_REQUESTED** [review]
Thanks for the new page! I've left some comments below for you to review.

File: pages/linux/exif.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`exif --extract-thumbnail --output={{thumbnail.jpg}} {{image.jpg}}`
+
+- Show the raw contents of the "Model" tag in the given image:
+
+`exif --ifd=0 --tag=Model --machine-readable {{image.jpg}}`
+
+- Change the value of the "Artist" tag to John Smith and save to new.jpg:
+
+`exif --output=new.jpg --ifd=0 --tag=Artist --set-value="John Smith" --no-fixup {{image.jpg}}`
```
[the remark this answers is not in the document]
Comment (ThatWeirdAndrew) [reply pages/linux/exif.md:24]: Should new.jpg and --ifd=0 be surrounded in curly brackets as well?

[the remark this answers is not in the document]
Comment (sbrl) [reply pages/linux/exif.md:24]: Ah, I missed that! Nice catch - suggestions updated.

**Comment by ThatWeirdAndrew** [comment]
Alright, I've edited the file so it should be good now.

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @ThatWeirdAndrew welcome and thanks for your submission. I left some comments.

**Comment by ThatWeirdAndrew** [comment]
Done

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @ThatWeirdAndrew!

**Pull request merged**